app/frontend/src/features/messages/groupchats/GroupChatView.tsx

```diff
@@ -0,0 +1,65 @@
+import { Box, BoxProps, IconButton, Typography } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import CloseIcon from "@material-ui/icons/Close";
+import * as React from "react";
+import { useEffect, useState } from "react";
+import Alert from "../../../components/Alert";
+import CircularProgress from "../../../components/CircularProgress";
+import { GroupChat, Message } from "../../../pb/conversations_pb";
+import { service } from "../../../service";
+import MessageList from "../messagelist/MessageList";
+
+const useStyles = makeStyles({ root: {} });
+
+export interface GroupChatViewProps extends BoxProps {
+  groupChat: GroupChat.AsObject;
+  handleClose: () => void;
+}
+
+export default function GroupChatView({
+  groupChat,
+  handleClose,
+}: GroupChatViewProps) {
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState("");
+  const [messages, setMessages] = useState<Message.AsObject[]>([]);
+
+  const handleSend = async (text: string) => {
+    await service.conversations.sendMessage(groupChat.groupChatId, text);
+    await fetchMessages();
```

Comment:
This might be difficult once pagination/scroll to load is implemented. But probably better to save that for later.

Reply:
Yes.